build.gradle

```diff
@@ -22,6 +22,7 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
     implementation 'org.jetbrains:annotations:20.1.0'
+    testImplementation 'junit:junit:4.13.1'
```

Comment:
kako to da koristiš verziju 4.x, postoji 5, pa ako baš nemaš razlog bolje je koristiti zadnu verziju. 
Također bolje je prvo probati bez verzije, samo `testImplementation 'junit:junit'` , na taj način će se povući verzija koja je postavljena u spring bootu, i onda znaš da je kompatibilna sa ostatkom spring boota. Ako vez verzije Intellij vrati grešku, to znači da verzija za taj library nije postavljena u Spring bootu i da moraš eksplicitno postaviti verziju